maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -220,6 +220,8 @@ class IOSDriver(
 
     override fun backPress() {}
 
+    override fun hideKeyboard() {}
```

Comment:
I just checked and it seems that pressing Escape key (key code `41`) on iOS is hiding the keyboard (at least in Safari). Let's use that

Reply:
updated on [27539f2](https://github.com/mobile-dev-inc/maestro/pull/165/commits/27539f2b48a65a80b12a6e3a2cff4b452ef92650)